bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/ladda/LaddaAjaxLink.java

```diff
@@ -78,23 +78,11 @@ protected void onInitialize() {
      * @param color The color for the spinner
      * @return {@code this}, for chaining
      */
-    public LaddaAjaxLink<T> setSpinnerColor(String color) {
+    public LaddaAjaxLink<T> setSpinnerColor(LaddaBehavior.Color color) {
         this.laddaBehavior.withSpinnerColor(color);
         return this;
     }
 
-    /**
-     * Sets the size of the spinner in pixels
-     *
-     * @param size The size of the spinner in pixels
-     * @return {@code this}, for chaining
-     */
-    public LaddaAjaxLink<T> setSpinnerSize(int size) {
```

Comment:
Why this is being removed ?
Maybe it should be reimplemented with an enum for:
```
data-size="xs"
data-size="s"
data-size="l"	
``` 
?

Reply:
the only size option https://getbootstrap.com/docs/4.4/components/spinners/#size is `sm`
all other customisations are possible via CSS
I don't want to add possibility to add inline CSS to comply with strict CSP
Ca add the option to add custom CSS class ...